modules/installation-osp-deploying-all-bare-metal-preexisting.adoc

```diff
@@ -0,0 +1,70 @@
+[id="installation-osp-deploying-all-bare-metal-preexisting_{context}"]
+= Configuring a cluster to use bare-metal control plane and compute machines
+
+If you want your cluster to use bare-metal control plane and compute machines, modify the
+`install-config.yaml` file. In this configuration, control plane and compute machines are
+deployed on user-provisioned infrastructure.
+
+[NOTE]
+====
+Bare-metal compute machines are not supported on clusters that use Kuryr.
+====
+
+.Prerequisites
+
+* The {rh-openstack} link:https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/16.1/html/bare_metal_provisioning/index[Bare Metal service (Ironic)] is enabled and accessible by using the {rh-openstack} Compute API.
+
+* Bare metal is available as link:https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/16.1/html/bare_metal_provisioning/sect-configure#creating_the_bare_metal_flavor[an {rh-openstack} flavor].
+
+* {rh-openstack} is running on network infrastructure that you provisioned.
+
+* The {rh-openstack} network supports bare metal server attachment.
+
+* You created an `install-config.yaml` file as part of the {product-title} installation process.
+
+.Procedure
+
+. In the `install-config.yaml` file, edit the flavors for `compute.platform.openstack.type` and `controlPlane.platform.openstack.type`, and enter a value for `platform.openstack.machinesSubnet`:
++
+.An example bare metal `install-config.yaml` file
+[source,yaml]
+----
+                controlPlane:
+                   platform:
+                     openstack:
+                       type: my-bare-metal-flavor <1>
+                 ...
+
+                 compute:
+                 - architecture: amd64
+                   hyperthreading: Enabled
+                   name: worker
+                   platform:
+                     openstack:
+                       type: my-bare-metal-flavor <1>
+                   replicas: 3
+                 ...
+                 platform:
+                   openstack:
+                     machinesSubnet: 25495f6a-8eae-43ff-ad7b-77ba57ed0a04 <3>
+----
+<1> Change these values to a bare metal flavor.
+<2> Change this value to the UUID of the {rh-openstack} subnet that is already provisioned.
+
```

Comment:
Also missing:
```
<3> Change this value to the UUID of the {rh-openstack} subnet that is already provisioned.
```

Reply:
Did you mean <2>, there? Verify my next commit. @mandre

Reply:
https://github.com/openshift/openshift-docs/pull/25167/commits/e2e8bf4c306907d06d3ac71a768a4cb4942d47e5